server/src/main/java/org/elasticsearch/action/admin/cluster/stats/AnalysisStats.java

```diff
@@ -118,6 +128,15 @@ public static AnalysisStats of(Metadata metadata, Runnable ensureNotCancelled) {
             Map<String, Settings> tokenFilterSettings = indexSettings.getGroups("index.analysis.filter");
             usedBuiltInTokenFilters.keySet().removeAll(tokenFilterSettings.keySet());
             aggregateAnalysisTypes(tokenFilterSettings.values(), usedTokenFilterTypes, indexTokenFilterTypes);
+            if (SynonymsAPI.isEnabled()) {
+                aggregateSynonymFilterTypes(
+                    tokenFilterSettings.values(),
+                    usedSynonymSets,
+                    "synonyms_set",
```

Comment:
Should this hardcoded string be moved into a variable?

Reply:
I was thinking on the follow-up PR for this, in which we will include `synonyms_path` - but removing it for now 👍